server/hap2/hatohol/hap2_nagios_ndoutils.py

```diff
@@ -117,6 +118,17 @@ def __parse_url(self, url):
 
         return server, port, database
 
+    def __convert_to_nagios_time(self, hatohol_time):
+        over_second= hatohol_time.split(".")[0]
+        time_array = [char for char in over_second]
+        time_array.insert(4, "-")
+        time_array.insert(7, "-")
+        time_array.insert(10, " ")
+        time_array.insert(13, ":")
+        time_array.insert(16, ":")
+
```

Comment:
>  it looks this function is only used as making a last_info. 

That's wrong.
It used to create SQL sentence. 
datetime object can not use sql.
So, we should use str() as last_info.